CAP/gap/CategoryMorphisms.gd

```diff
@@ -1164,10 +1164,9 @@ DeclareOperation( "AddIsColiftableAlongEpimorphism",
                   [ IsCapCategory, IsList ] );
 
 #! @Description
-#! The arguments are two morphisms $\alpha: a \rightarrow c$, $\beta: b \rightarrow c$
-#! such that there is a lift $\alpha / \beta: a \rightarrow b$ of $\alpha$ along $\beta$, i.e.,
-#! a morphism such that $\beta \circ (\alpha / \beta) \sim_{a,c} \alpha$.
-#! The output is such a lift or $\mathtt{fail}$ if it doesn't exist.
+#! The arguments are two morphisms $\alpha: a \rightarrow c$, $\beta: b \rightarrow c$.
+#! The output is a lift $\alpha / \beta: a \rightarrow b$ of $\alpha$ along $\beta$
```

Comment:
Could you leave the explanation of what a lift is (the part in the original documentation after the` i.e.`). I find it helpful whenever I look up `Lift/Colift`.

Same for colifts.

Reply:
Done